plugins/kubevirt/templates/installer.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .Release.Name }}-kubevirt
+  namespace: argocd
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+    - post-delete-finalizer.argocd.argoproj.io
+    - post-delete-finalizer.argocd.argoproj.io/cleanup
+spec:
+  project: default
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: kubevirt
+  sources:
+    - path: plugins/kubevirt/releases/{{ .Values.version }}/
+      repoURL: https://github.com/juno-fx/Terra-Official-Plugins.git
```

Comment:
This should be dynamic for forked repos, while the users could hardcode it to their repo once forked, I think it be more clear and provide more flexibility to have this as a field they can input.